archive.php

```diff
@@ -5,7 +5,6 @@
 <div class="container">
 	<div class="row">
 		<div class="col-md-8">
```

Comment:
`col-lg-8` would be ideal here for consistency with other two column layouts in the new theme; the -md breakpoint starts to get a bit scrunched

Reply:
Updated.